model_compression_toolkit/core/common/quantization/quantization_params_generation/power_of_two_selection.py

```diff
@@ -55,8 +55,9 @@ def power_of_two_selection_tensor(tensor_data: np.ndarray,
         threshold = max_power_of_two(tensor_max, min_threshold)
     else:
         signed = True  # weights are always signed
+        axis = -1 if per_channel else None
```

Comment:
I don't think this is the case for Pytorch. I think you need to use the fw_info

Reply:
Numpy. This is in the common section.